src/viewer.js

```diff
@@ -364,6 +366,20 @@ $.Viewer = function( options ) {
         viewer: this
     });
 
+    /**
```

Comment:
Sorry for the confusion about the doc comments. We don't need a doc comment here; please remove it and the one for `remove-item`.